exemplar/README.md

```diff
@@ -1,21 +1,48 @@
 # Exemplar 
-This is a sample app that makes use of MISK.
+This is a sample app that makes use of MISK. All commands below assume you're in the root directory 
+of this repository.
 
 ## Building
 Build exemplar:
 
 ```
-  $ ../gradlew clean jar
+  $ ./gradlew clean jar
 ```  
 
 ## Docker
-Run exemplar in Docker:
+### Building
+Build a Docker image of Exemplar:
 
 ```
-  $ docker build -t exemplar-0.0.1 .
-  $ docker run -p 8080:8080 exemplar-0.0.1
+  $ docker build -t exemplar-0.0.1 exemplar/  
 ```
 
+### Running locally
 Visit [Docker for Mac](https://docs.docker.com/docker-for-mac/install/) to install Docker on a Mac for testing.
 
+Run exemplar in Docker locally:
+```
+  $ docker run -p 8080:8080 exemplar-0.0.1
+```
+
+### Running on GCP
+
+#### Tag your image and push to GCR
+```
+  $ docker tag misk-exemplar:0.0.1 gcr.io/${YOUR_GCP_PROJECT}/misk-exemplar:0.0.1
+  $ gcloud docker -- push gcr.io/${YOUR_GCP_PROJECT}/misk-exemplar:0.0.1
+```
+
+#### Start a Kubernetes Cluster (if you don't already have one)
```

Comment:
Possibly mention this will cost money? I think that's pretty standard, unless t can run on a free tier.